pkg/server/file.go

```diff
@@ -114,6 +114,9 @@ func (t *TranslateServiceServer) UploadTranslationFile(
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
 
+	// Original status for translation is taken from request parameters,
+	translation.Original = params.original
```

Comment:
redundant?

`TranslationFromData()` takes params, and creates translation with the original flag set to correct value from params, or file itself if possible,
[code](https://github.com/translate-agent/translate/blob/4fb9df5bde269292ab753dc7cd761e8f4459fb91/pkg/server/convert.go#L43C2-L43C56)

Reply:
I added it there because tests were failing, will look into it.

Reply:
the only case that could fail could be xliff, because it is the only one that can be determined from source file, afaik

Reply:
Shouldn't we ignore what's in the file? What matters should be parameter passed in through request.